docs/quick-install.md

```diff
@@ -1,175 +1,184 @@
+    sudo signalfx-agent status
+    ```
 
+    The return should be similar to the following example:  
 
-### Step 2. Confirm your Installation 
+    ```sh
+    SignalFx Agent version:           4.7.6
+    Agent uptime:                     8m44s
+    Observers active:                 host
+    Active Monitors:                  16
+    Configured Monitors:              33
+    Discovered Endpoint Count:        6
+    Bad Monitor Config:               None
+    Global Dimensions:                {host: my-host-1}
+    Datapoints sent (last minute):    1614
+    Events Sent (last minute):        0
+    Trace Spans Sent (last minute):   0
+    ```
 
-To confirm the SignalFx Smart Agent installation is functional on either platform, enter:
+2. Enter the following commands to verify the installation:
 
-```sh
-sudo signalfx-agent status
-```
+    | Command | Description   |
+    |---|---|
+    | <code>signalfx-agent status config</code>   | This command shows resolved config in use by the Smart Agent. |
+    | <code>signalfx-agent status endpoints</code>  | This command shows discovered endpoints.  |
+    | <code>signalfx-agent status monitors</code>  | This command shows active monitors.  |
+    | <code>signalfx-agent status all</code>  | This command shows all of the above statuses. |
 
-The response you will see is similar to the one below:
+***
 
-```sh
-SignalFx Agent version:           4.7.6
-Agent uptime:                     8m44s
-Observers active:                 host
-Active Monitors:                  16
-Configured Monitors:              33
-Discovered Endpoint Count:        6
-Bad Monitor Config:               None
-Global Dimensions:                {host: my-host-1}
-Datapoints sent (last minute):    1614
-Events Sent (last minute):        0
-Trace Spans Sent (last minute):   0
-```
+### Troubleshoot the Smart Agent installation
 
-To verify the installation, you can run the following commands:
+If you are unable to install the Smart Agent, consider the following issues:
 
-```sh
-signalfx-agent status config - show resolved config in use by agent
-signalfx-agent status endpoints - show discovered endpoints
-signalfx-agent status monitors - show active monitors
-signalfx-agent status all - show everything
-```
+***
+ 
+#### Update your realm
+
+By default, the Smart Agent will send data to the *us0* realm. As a result, you may need to update the realm used in the installation process. To locate your realm: 
 
-#### Troubleshoot any discrepancies in the Installation 
+1. In SignalFx, in the top, right corner, click your profile icon.
+2. Click **My Profile**.
+3. Next to **Organizations**, review the listed realm.
 
-##### Realm 
+***
 
-By default, the Smart Agent will send data to the us0 realm. If you are not in this realm, you will need to explicitly set the signalFxRealm option in your config like this:
+#### Set the endpoints
 
+To explicitly set the ingest, API server, and trace endpoint URLs, review the following examples:  
```

Comment:
Same comment as above

Reply:
You're right. It's not overly clear how configuring this information will help the installation process, let alone how to configure this information. Maybe we should consider removing it completely?

Reply:
Yep, this also goes.

Reply:
This section has been removed.